heed/src/env.rs

```diff
@@ -24,7 +24,13 @@ use crate::mdb::ffi;
 /// noone tries to open the same environment between these two phases.
 ///
 /// Trying to open a None marked environment returns an error to the user trying to open it.
-static OPENED_ENV: Lazy<RwLock<HashMap<PathBuf, (Option<Env>, Arc<SignalEvent>)>>> = Lazy::new(RwLock::default);
+static OPENED_ENV: Lazy<RwLock<HashMap<PathBuf, EnvCacheEntry>>> = Lazy::new(RwLock::default);
+
+struct EnvCacheEntry {
```

Comment:
This is not a cache entry in the sense that it is required for the library to work and to be safely used and that it can't be evicted from this `HashMap` after a certain time of anything other than closing the env itself.

```suggestion
static OPENED_ENV: Lazy<RwLock<HashMap<PathBuf, EnvEntry>>> = Lazy::new(RwLock::default);

struct EnvEntry {
```